docs/deployment/plugins/k8s/index.rst

```diff
@@ -431,7 +431,20 @@ Specify plugin configuration
                             - kind: ServiceAccount
                               name: spark
                               namespace: "{{ namespace }}"
-
+                  rbac:
```

Comment:
```suggestion
```

Reply:
@aazam-gh Please remove this whole section as it belongs to Spark config, which is not the focus of the issue